docs/Operator guide/Empty tiles.md

```diff
@@ -10,7 +10,7 @@ Further investigation should also be done on `OATiles-hint: empty` response head
 
 LINZ has imagery datasets which are very sparse, with large empty areas between smaller sections of valid data, for example [Gisborne Urban 2022-2023](https://basemaps.linz.govt.nz/?i=gisborne-2022-2023-0.1m).
 
-[![Gisborne 2023](./static/2023-06-26-gisborne-2023.png)](https://basemaps.linz.govt.nz/?i=gisborne-2022-2023-0.1m)
+[![Gisborne 2023](./images/2023-06-26-gisborne-2023.png)](https://basemaps.linz.govt.nz/?i=gisborne-2022-2023-0.1m)
```

Comment:
I would prefer this to be left as static to be consisetent with landing and also if we want to put other static items into it, eg example tar or config files.

Reply:
Agreed, have renamed to `static`.